README.md

```diff
@@ -233,32 +250,35 @@ $ tree my-component
 First, install the cli and then use `manifest-gen` to do this.
 
 ``` shell
-$ cd modality-probe-cli
+$ cd ../modality-probe-cli
```

Comment:
Why the `..`? The commands as written assume you're in the root of the repo, which is standard

Reply:
:+1: This is inherited from the original `modality-probe-capi` README.md; I couldn't tell you why. Changing to:
`$ cd modality-probe-cli`